src/Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServer.cs

```diff
@@ -29,6 +30,7 @@ public class KestrelServer : IServer
         private readonly ITransportFactory _transportFactory;
 
         private bool _isRunning;
+        private int _isStopping;
```

Comment:
Nit: Since this remains set after StopAsync completes, I prefer calling this _stopped instead of _isStopping.